### PR TITLE
add user detail as default in call for speaker form

### LINKED ISSUE #773: Use personal profile details and avatar as speaker details by default
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
Currently to add speaker details, you need to separately add a speaker profile and details. This is confusing since you already have a personal profile with avatar in the top right, which is not reflected in the »speakers« list of an event.

**Expected behavior:**
By default, simply take over the info from the personal profile. Editing the speaker profile per event should only be optional.

### PR DESCRIPTION
**Pull request by k4ni5h**
session-speaker-form.hbs: add condition and user data
This change add user data as default for the speaker detail
Fixes #773

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-975-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-975-open-event-frontend-fossasia.surge.sh

File: app/components/forms/session-speaker-form.js [hunk 1]
```diff
@@ -311,7 +311,9 @@ export default Component.extend(FormMixin, {
       this.set('data.speaker', this.get('data.speaker').toArray()[0]);
     } else if (this.get('isCFS')) {
       this.set('data.speaker', this.get('store').createRecord('speaker', {
-        email : this.get('data.currentUser.email'),
+        email : this.get('authManager.currentUser.email'),
```
Comment (geekyd) [inline app/components/forms/session-speaker-form.js]: @k4ni5h Please fix lint errors. Before pushing the code or sending a PR please make sure that the tests are passing on your local machine. :)

Nice work 👍

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "broccoli-asset-rev": "^2.4.5",
```
Comment (geekyd) [inline package.json]: @k4ni5h Please revert these changes

**Comment by niranjan94** [comment]
Deployment link: https://pr-975-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@k4ni5h Revert changes made in package.json & package-lock.json

**Comment by niranjan94** [comment]
Deployment link: https://pr-975-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-975-open-event-frontend-fossasia.surge.sh
